source/beamline/CLS/CLSValves.h

```diff
@@ -16,6 +23,9 @@ class CLSValves : public CLSTriStateGroup
 	/// Destructor.
 	virtual ~CLSValves();
 
+	/// helper function to create a beam on action list
+	AMListAction3* createBeamOnActionList();
+
```

Comment:
@dretrex we never turn Valves off when we turning beam off.
Also in the following implementation, it also mentioned that we don't support close valves
```
AMAction3* CLSValves::createMoveAction(double setpoint)
{
	AMAction3 *result = 0;

	if (setpoint == Open)
		result = createMoveToOpenAction();

	// Closing valves is not supported at this time.
	// This is because a user accidentally closing valves
	// could trigger machine protection protocols.

	return result;
}
```